packages/cli/.nycrc

```diff
@@ -19,6 +19,5 @@
   ],
   "all": true,
   "cache": true,
-  "extends": "@istanbuljs/nyc-config-typescript",
```

Comment:
Is there are reason why we are removing this extends? nyc recommends using this (https://github.com/istanbuljs/nyc?tab=readme-ov-file#typescript-projects)